relayer/pkg/chainlink/ocr2/utils.go

```diff
@@ -221,7 +221,7 @@ func parseConfigEventData(eventData []*caigotypes.Felt) (types.ContractConfig, e
 		Min: signedFelt(onchainConfigFelts[1]),
 		Max: signedFelt(onchainConfigFelts[2]),
 	}
-	onchainConfig, err := temp.Encode()
+	onchainConfig, err := median.StandardOnchainConfigCodec{}.Encode(temp)
```

Comment:
@archseer how were you thinking about using a custom onchainConfig codec here? would that involve contract changes so version/min/max were encoded together?